### ISSUE TITLE
Start Leveraging Open UI

### ISSUE THREAD

**Issue by mgifford** [uswds/uswds#4748]
Seems that it would make sense in future releases of the USWDS to plan to build around something like [Open UI](https://open-ui.org/).

The more we can move to just styling generic components, the easier it should be to address accessibility and performance issues that currently plague many government sites.
